ingestion/docker-compose.yml

```diff
@@ -1,12 +1,12 @@
 web:
     build: .
-    command: python -u ingestion/run.py
+    command: python -u run.py
```

Comment:
remove `ingestion/docker-compose.yml`
create `/docker-compose.yml` that brings up all the services